src/NHSD.BuyingCatalogue.Identity.Api/Controllers/ConsentController.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using System.Globalization;
 using System.Threading.Tasks;
 using IdentityServer4.Extensions;
 using Microsoft.AspNetCore.Authorization;
```

Comment:
Aware it's not been added as part of this PR but do we need `using Microsoft.AspNetCore.Authorization;`?

Reply:
removed

Reply:
done